README.md

```diff
@@ -473,15 +473,27 @@ directory. It's similar to the `COPY` command in a Dockerfile.
 Params:
 
 - `paths`: a list of files and/or directories to copy. These may use template
-  expressions (e.g. `{{.my_input}}`). Directories will be crawled recursively
+  expressions or simple file globs (e.g. `{{.my_input}}`, `*.txt`). Directories will be crawled recursively
```

Comment:
prettier changed everything from single to double quotes, I think it might've been https://marketplace.visualstudio.com/items?itemName=stkb.rewrap (I had this installed from one of our pair programming sessions)

Reply:
re-formatted.